tests/all.sh

```diff
@@ -55,3 +50,28 @@ RUST_BACKTRACE=1 RUST_LOG=info cargo test -- --test-threads=1 --nocapture
 # Stop TPM server #
 ###################
 pkill tpm_server
+
+#############################
+# Install nightly toolchain #
+#############################
+rustup toolchain install nightly
+
+############################
+# Install legacy toolchain #
+############################
+rustup toolchain install 1.38.0
+
+####################
+# Verify doc build #
+####################
+cargo +nightly doc --features docs --verbose
```

Comment:
Would that command fail if any of the dependency fail to build on nightly? Does docs.rs build with `--no-deps`?

Reply:
... I started making this change and then got carried away trying to do something else, forgot about actually adding that. Will do